packages/expo-updates/src/Updates.ts

```diff
@@ -169,6 +170,41 @@ export async function checkForUpdateAsync(): Promise<UpdateCheckResult> {
   return result;
 }
 
+/**
+ * Retrieves the most recent expo-updates log entries from the client.
+ *
+ * @param maxAge Sets the max age of retrieved log entries in milliseconds. Default to 3600000 ms (1 hour).
+ *
+ * @return A promise that fulfills with an array of [`UpdatesLogEntry`](#updateslogentry) objects;
+ *
+ * The promise rejects if there is an unexpected error in retrieving the logs.
+ */
+export async function readLogEntriesAsync(maxAge: number = 3600000): Promise<UpdatesLogEntry[]> {
```

Comment:
if this is always defaulted then no need to handle defaults in native code (can require the arg in native code)

Reply:
Done